src/test/ui/parser/raw/raw-str-in-macro-call.rs

```diff
@@ -0,0 +1,9 @@
+// check-pass
+
+macro_rules! m {
+    ($tt:tt #) => ()
+}
+
+fn main() {
+    m!(r#"abc"##);
+}
```

Comment:
Can we add another tests with the following combinations?

```
macro_rules! m {
    ($tt:tt) => ()
}

 fn main() {
    m!(r#"abc"##);
}
```
```
macro_rules! m {
    ($tt:tt #) => ()
}

 fn main() {
    m!(r#"abc"#);
}
```

Just want us to catch any regressions in the future.